manager.go

```diff
@@ -194,11 +197,83 @@ func (m *Manager) GetStats() (Stats, error) {
 
 	stats.Processed = storeStats.Processed
 	stats.Failed = storeStats.Failed
-	stats.Retries = storeStats.Retries
+	stats.RetryCount = storeStats.RetryCount
 
 	for q, l := range stats.Enqueued {
 		stats.Enqueued[q] = l
 	}
 
 	return stats, nil
 }
+
+// GetRetries returns the set of retry jobs for the manager
+func (m *Manager) GetRetries() (Retries, error) {
+	retries := Retries{}
+
+	storeRetries, err := m.opts.store.GetAllRetries()
+	if err != nil {
+		return retries, err
+	}
+
+	retryStats, err := m.opts.client.ZRange(m.opts.Namespace+storage.RetryKey, 0, 1).Result()
```

Comment:
This number needs to be retrieved via `store.`

Reply:
No longer using `ZRange`